bundles/CoreBundle/Command/ThumbnailsImageCommand.php

```diff
@@ -149,54 +169,82 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         $list = new Asset\Listing();
         $list->setCondition(implode(' AND ', $conditions));
-        $total = $list->getTotalCount();
-        $perLoop = 10;
 
-        $totalToGenerate = $total * count($thumbnailsToGenerate);
+        $items = [];
```

Comment:
I think we need to use `loadIdList()` here to avoid memory issues when having hundreds of thousands assets to process.

Reply:
done!